docs/requirements/windows.md

```diff
@@ -24,7 +24,11 @@ Ruby is used solely to build SCSS
 ```shell
 choco install ruby --confirm
 chcp 1252
-gem source -a http://rubygems.org/
+```
+
+#### Restart CMD before proceding to the next commands
+```shell
+gem source -a https://rubygems.org/
```

Comment:
Here we want to add the `http` as a fallback to the default `https` because we something have glitches on Windows

Reply:
Ok seems to be the way to do it now.